fetch/api/resources/basic.html

```diff
@@ -0,0 +1 @@
+<!DOCTYPE html>
```

Comment:
I don't think tests run concurrently so if you clean up that could work, but it might be clearer to just add a comment as to why you duplicated and keep the scope as-is.